libmei/atttypes.h

```diff
@@ -318,6 +318,12 @@ enum data_HEADSHAPE_list {
     HEADSHAPE_list_slash,
     HEADSHAPE_list_square,
     HEADSHAPE_list_x,
+    
+    /* The following SMuFL glyphs are not part of MEI's data.HEADSHAPE.list */
+    HEADSHAPE_list_circlex,
```

Comment:
I saw that you raised the issue over at at music-encoding (https://github.com/music-encoding/music-encoding/issues/400), thanks.